pkg/distributor/distributor.go

```diff
@@ -421,7 +421,7 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 		incomingRequests: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
 			Name: "cortex_distributor_requests_in_total",
 			Help: "The total number of requests that have come in to the distributor, including rejected or deduped requests.",
-		}, []string{"user"}),
+		}, []string{"user", "version"}),
```

Comment:
I wonder whether we should have two metrics instead with constant labels. Maybe slightly more efficient? Although we already have a dynamic attribute in "user" . Also we might have more detailed version later?

Reply:
I don't know personally.